libethashseal/GenesisInfo.cpp

```diff
@@ -77,6 +78,8 @@ std::string const& dev::eth::genesisInfo(Network _n)
         return c_genesisInfoExperimentalTransitionTest;
     case Network::ConstantinopleFixTest:
         return c_genesisInfoConstantinopleFixTest;
+    case Network::IstanbulTransitionTest:
```

Comment:
Do we need to add configuration information for Instanbul as a part of this PR?

Reply:
I'll split it into separate PR. (note that this is a config only for tests, though)